### PR TITLE
LT-1120: Fixed warnings for packages version and misusage

### PR DESCRIPTION
**Pull request by rodrigueseder**

### CODE REVIEW

File: src/Chest/Chest.csproj [hunk 1]
```diff
@@ -49,7 +49,7 @@
     <PackageReference Include="Lykke.Common.ApiLibrary" Version="1.7.1" />
     <PackageReference Include="Lykke.Middlewares" Version="1.0.0-1537228216" />
     <PackageReference Include="Lykke.Snow.Common.Startup" Version="1.2.2" />
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.1.5" />
+    <PackageReference Include="Microsoft.AspNetCore.All" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Versioning.ApiExplorer" Version="2.2.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.0-beta2" />
```
Comment (Waqas-Zamir) [inline src/Chest/Chest.csproj]: this should be updated to `2.6.2` as well

Comment (dafigerdjikova) [reply src/Chest/Chest.csproj]: there is no such version @Waqas-Zamir

Comment (Waqas-Zamir) [reply src/Chest/Chest.csproj]: actually it is a beta version and a stable version is available.
and we are using that version in most of the other Lego projects.
https://bitbucket.org/lykke-snow/cronut/src/bcb37bd1888e069bf8f4af4d1ef61e7eb7fe7055/src/Cronut/Cronut.csproj#lines-51

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+## Unreleased
```
Comment (Waqas-Zamir) [inline CHANGELOG.md]: if we want to release it asap then please change the Unreleased to version `2.11.1` or revert the .csproj version :)

Comment (dafigerdjikova) [reply CHANGELOG.md]: We won't release it :) maybe with next release

File: src/Chest/Chest.csproj (hunk 1 above)
Comment (dafigerdjikova) [reply src/Chest/Chest.csproj]: done, I was checking wrong url

**Pull request merged**